blazor/datagrid/how-to/resolve-the-issue-of-two-default-validation-messages-shown-in-grid-default-dialog-editor.md

```diff
@@ -0,0 +1,130 @@
+---
+layout: post
+title: Resolve the issue of two default validation messages shown in Grid default dialog editor in Blazor DataGrid Component | Syncfusion
+description: Learn here all about resolve the issue of two default validation messages shown in Grid default dialog editor
+platform: Blazor
+control: DataGrid
+documentation: ug
+---
+
+# Resolve the issue of two default validation messages shown in Grid default dialog editor:
+
+**Error screenshot**
+
+![Issue of two default validation messages shown in Grid default dialog editor](../images/blazor-grid-issue-of-two-default-validation-messages.png)
+
+By default, the DataGrid uses two validator components. This issue occurs because the DataGrid uses both data annotation enabled and the ValidationMessage property defined in DataGrid component, which can lead to the issue mentioned above. To resolve this issue, disable the built-in validator and achieve column validation solely using Data Annotations or wise versa.
```

Comment:
his issue occurs because the DataGrid uses both data annotation enabled and the ValidationMessage property defined in DataGrid component, which can lead to the issue mentioned above ---- change this to "This issue occurs because the DataGrid uses both data annotation enabled and the ValidationMessage property defined in DataGrid component's Template"